### ISSUE TITLE
[v2] crash when my credentials are not entered

### ISSUE THREAD

**Issue by andybalaam** [cerisara/DragonGoApp#11]
Hi, I installed the apk from http://talc1.loria.fr/users/cerisara/DragonGoAppv2.apk

Before I had entered any DGS credentials, I tapped the "..." button in the top right.

The application crashed.

**Comment by cerisara** [comment]
Sorry, but I couldn't reproduce...
I just tried a fresh install on my phone and tapped "..." first: the menu appeared normally.
No crash (?)
After a fresh install, I waited until it has finished copying eidogo, then I saw a minor bug: the goban did not draw, but some sgf text were printed instead (just as in your previous issue, I guess). But in V2, there's this "refresh" button: I just pressed it and the goban redrawed nicely.
Still, no crash after "...", just the expected menu.

It's frustrating not to be able to reproduce bugs ;-)
